### PR TITLE
add deduplication script

### PR DESCRIPTION
**Pull request by mspector**

### CODE REVIEW

File: venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+# iterate through the duplicates
+for paperhash, notes in duplicates_by_paperhash.iteritems():
+
+    # for every paperhash, sort the notes by tcdate.
+    # the first note should be the one posted last.
+    latest_note = sorted(notes, key=lambda n: n.tcdate, reverse=True)[0]
+    assert latest_note.tcdate == max([n.tcdate for n in notes])
+
+    print "keeping ",latest_note.id
```
Comment (melisabok) [inline venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py]: I thought you wanted to keep the first submission?

File: venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+            try:
+                deleted_n = client.post_note(n)
+                print "--> deleting original ", deleted_n.id
+            except openreview.OpenReviewException as e:
+                print 'ERROR on {0}:{1} '.format(n.id, e)
+
+
+            try:
+                blind_ref = blindrefs_by_original[n.id]
```
Comment (melisabok) [inline venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py]: why are you deleting the reference and not the blinded note?

File: venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                blind_ref.ddate = 1509138000000
+                deleted_ref = client.post_note(blind_ref)
```
Comment (melisabok) [inline venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py]: I'm not sure that deleting the reference will delete the note

File: venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py (hunk 1 above)
Comment (mspector) [reply venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py]: No, I think the last one is better. Sometimes people submitted twice, thinking the second would overwrite the first

File: venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py (hunk 2 above)
Comment (mspector) [reply venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py]: I thought this was the proper way to do it, but I'll check again, maybe you are right

File: venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py (hunk 3 above)
Comment (melisabok) [reply venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py]: I checked the backed and we you update a reference with the same id as the note then we do an update note, so I'm not sure what is the content of the blinded note after this, could you check?

File: venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py (hunk 2 above)
Comment (melisabok) [reply venues/ICLR.cc/2018/Conference/python/deduplicate-submissions.py]: I think you have to update the blinded note instead of the reference.

**Pull request merged**